etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -95,13 +103,34 @@ spec:
         - mountPath: /pg-ca-cert
           name: pg-ca-cert
         {{- end }}
+      {{- if .Values.global.securityContexts.enabled }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          capabilities:
+              drop:
+              - all
+      securityContext:
+        runAsUser: {{ .Values.global.securityContexts.runAsUser }}
```

Comment:
pg-bouncer user is `1001` so global `runAsUser` wont work here. I am assuming its `1000` globally.  https://github.com/bitnami/bitnami-docker-pgbouncer/blob/1.17.0-debian-11-r3/1/debian-11/Dockerfile#L38

Reply:
This does work. The container permissions have read and execute access for other, so any uid can be used.